Templating/FilterExtension.php

```diff
@@ -24,9 +24,9 @@ class FilterExtension extends AbstractExtension
     use FilterTrait;
 
     /**
-     * {@inheritdoc}
+     * @return TwigFilter[]
      */
-    public function getFilters()
+    public function getFilters(): array
```

Comment:
this is technically a BC break, in case somebody extends this class and overwrites getFilters. it sounds like a weird idea to extend the class, which is why we made the new LazyFilterExtension final.

as its deprecated anyways, i'd rather not change the method signature here.

Reply:
oops sorry, I thought I only changed `LazyFilterExtension`, this file was not meant to be changed.